src/Ucommerce.Avenue.Umbraco/Controllers/CategoryController.cs

```diff
@@ -1,49 +1,46 @@
 ﻿using System.Collections.Generic;
 using System.Web.Mvc;
-using UCommerce.Api;
+using Ucommerce.Api;
+using Ucommerce.Api.Search;
 using UCommerce.Content;
-using UCommerce.EntitiesV2;
 using Umbraco.Web.Mvc;
-using UCommerce.Extensions;
 using UCommerce.Infrastructure;
 using UCommerce.RazorStore.Models;
-using UCommerce.Runtime;
-using UCommerce.Search.Facets;
+using UCommerce.Search;
+using UCommerce.Search.FacetsV2;
+using UCommerce.Search.Models;
 using Umbraco.Web.Models;
 
 namespace UCommerce.RazorStore.Controllers
 {
     public class CategoryController : RenderMvcController
     {
+        private readonly ICatalogLibrary _catalogLibrary = ObjectFactory.Instance.Resolve<ICatalogLibrary>();
+        private readonly ISiteContext _siteContext = ObjectFactory.Instance.Resolve<ISiteContext>();
+        private readonly ISearchLibrary _searchLibrary = ObjectFactory.Instance.Resolve<ISearchLibrary>();
+
```

Comment:
@skovsboll Should we do
A) private readonly ICatalogContext _catalogContext
B) public ICatalogContext CatalogContext
?

Let's pick one and be consistent with our naming :)

Reply:
I guess we go with A.
Good catch